packages/docusaurus/src/webpack/utils.ts

```diff
@@ -206,88 +184,50 @@ export function applyConfigurePostCss(
     }
   }
 
+  // @ts-expect-error @types/webpack strikes again
   config.module?.rules.forEach(overridePostCssOptions);
 
   return config;
 }
 
-// See https://webpack.js.org/configuration/stats/#statswarningsfilter
-// @slorber: note sure why we have to re-implement this logic
-// just know that legacy had this only partially implemented, so completed it
-type WarningFilter = string | RegExp | ((warning: string) => boolean);
-function filterWarnings(
```

Comment:
has the warning filter been implemented in another way? or can we just remove this as it's not useful anymore?